orchagent/bufferorch.cpp

```diff
@@ -1001,8 +1018,17 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
         // set order is detected.
         for (const auto &port_name : port_names)
         {
-            if (gPortsOrch->isPortAdminUp(port_name)) {
-                SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
+            if(local_port == true)
```

Comment:
why is this check needed? in L899, this check is already done and port_name is assigned as the local_port_name

Reply:
we are looping through port_names here hence port_name is getting assigned back to original name. Alos, during testing, i did see this behavior as well.